### PR TITLE
docs: updating createPaymentMethod params

### PR DESCRIPTION
**Pull request by vivekvardhanadepu**
## Testing

- [x] I tested this manually
- [ ] I added automated tests

## Documentation

Select one: 
- [ ] I have added relevant documentation for my changes.
- [ ] This PR does not result in any developer-facing changes.

### CODE REVIEW

File: docs/tipsi-stripe-migration-guide.md [hunk 1]
```diff
@@ -194,7 +194,7 @@ _after_:
 ```tsx
 // ...
 const { paymentMethod, error } = await createPaymentMethod({
-  type: 'Card',
+  paymentMethodType: 'Card',
```
Comment (charliecruzan-stripe) [inline docs/tipsi-stripe-migration-guide.md:197]:
```suggestion
  paymentMethodType: 'Card',
  paymentMethodData: {
    billingDetails: {
      email: 'email@stripe.com',
    }, // optional
  },
```

**Review by charliecruzan-stripe — CHANGES_REQUESTED** [review]
thanks for catching this! can you also change L287 and L288 to:

```
  paymentMethodType: 'Card',
  paymentMethodData: {
    billingDetails,
  },
  ```

**Review by charliecruzan-stripe — CHANGES_REQUESTED** [review]
can you also change L287 and L288 to:

```
  paymentMethodType: 'Card',
  paymentMethodData: {
    billingDetails,
  },
```

**Comment by vivekvardhanadepu** [comment]
> can you also change L287 and L288 to:
> 
> ```
>   paymentMethodType: 'Card',
>   paymentMethodData: {
>     billingDetails,
>   },
> ```

you mean in confirmSetupIntent?

**Review by charliecruzan-stripe — APPROVED** [review]
will merge once the CLA is signed :)

**Comment by vivekvardhanadepu** [comment]
> will merge once the CLA is signed :)

Done

**Pull request merged**